core/execution/common/market_activity_tracker.go

```diff
@@ -923,7 +923,7 @@ func (mt *marketTracker) getReturns(party string, windowSize int) ([]num.Decimal
 		if v, ok := epochData[party]; !ok {
 			returns = append(returns, num.DecimalZero())
 		} else {
-			returns = append(returns, num.MaxD(num.DecimalZero(), v))
+			returns = append(returns, v)
```

Comment:
nit-pick, I think we don't even need this `if-else` anymore, if `ok == false`, v should be `num.Decimal{}`, which is a nil/zero value

Reply:
good point. done.